### DISCUSSION TITLE
Put all assigments on one line

### DISCUSSION THREAD

**Discussion by mazegen** [uncrustify/uncrustify#4334] [Q&A]
There are specific options for assignments and new lines, for example `nl_assign_brace` for `= {`, but I need to make put all assignments on one line, is it possible? It's strange that are so many specific options but this general one seems to be missing (`nl_assign` doesn't work).

**Comment by guy-maurel** [comment]
please, create a new issue, provide your input file, the config and the output you expect to get

**Comment by mazegen** [comment]
Sure, here it is: https://github.com/uncrustify/uncrustify/issues/4340
